python/HeaderService/hscalc.py

```diff
@@ -49,14 +49,15 @@ def get_date_utc(timeStamp=None):
     -------
 
     t: type of astropy.datetime.Time object
-       The astropy.datetime.Time object with the timeStamp requested in UTC.
+       The astropy.datetime.Time object with the timeStamp requested
+       in the defined scale.
 
     """
 
     if timeStamp is None:
         t = Time.now()
     else:
-        t = Time(timeStamp, format="unix")
+        t = Time(timeStamp, format="unix", scale=scale)
```

Comment:
I am virtually certain this does not work, based on painful experience trying to get TAI as unix seconds from AstroPy. The problem is that Astropy always treats format="unix" as UTC, even if you specify a different scale. You can easily see this by creating two Times with format="unix" and different scales and observing that the difference is 0 seconds.

It does work as expected with other formats, so this is what I ended up doing in ts_ATMCSSimulator for a similar problem. Perhaps you can adapt it:
```
def curr_tai():
    """Get the current time as TAI in unix seconds.

    This is the same format used by the ATPtg CSC.
    """
    # unfortunately I can't just output curr_time.tai.unix
    # because that has the same value as curr_time.utc.unix
    curr_time = Time.now()
    tai_minus_utc = (curr_time.tai.mjd - curr_time.utc.mjd)*24*60*60
    return curr_time.utc.unix + tai_minus_utc
```

Reply:
It seems to be working for me, for example:
```
import astropy
import time
from astropy.time import Time 

timeStamp = time.time()    
Time(timeStamp, scale='utc', format="unix").isot                                                                        
'2019-08-01T21:09:16.625'

Time(timeStamp, scale='tai', format="unix").isot                                                                        '2019-08-01T21:09:53.625'
```

Did I miss anything?

Reply:
I guess it is expressing time in unix seconds (e.g. `.unix` instead of `.iso`) that shows the problem (the two outputs are identical in that case). If you are using the code to write ISO times then it should work fine. Thank you for checking.